lms/templates/dashboard.html

```diff
@@ -102,115 +102,116 @@
+            % endfor
 
-            % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
-              <a href="${marketing_link('COURSES')}">
-                ${_("Explore courses")}
-              </a>
-
-            %endif
-        </div>
-        % endif
+            </ul>
+          % else:
+            <div class="empty-dashboard-message">
+              <p>${_("You are not enrolled in any courses yet.")}</p>
 
-        % if staff_access and len(errored_courses) > 0:
-          <div id="course-errors">
-            <h2>${_("Course-loading errors")}</h2>
+              % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
+                <a href="${marketing_link('COURSES')}">
+                  ${_("Explore courses")}
+                </a>
 
-          % for course_dir, errors in errored_courses.items():
-             <h3>${course_dir}</h3>
-                 <ul>
-               % for (msg, err) in errors:
-                   <li>${msg}
-                     <ul><li><pre>${err}</pre></li></ul>
-                   </li>
-               % endfor
-                 </ul>
-          % endfor
+              %endif
           </div>
-        % endif
-    </div>
+          % endif
 
-      %if sidebar_account_activation_message:
-        <div class="sidebar-notification">
-          ${sidebar_account_activation_message | n, decode.utf8}
-        </div>
-      %endif
+          % if staff_access and len(errored_courses) > 0:
+            <div id="course-errors">
+              <h2>${_("Course-loading errors")}</h2>
 
-      % if settings.FEATURES.get('ENABLE_DASHBOARD_SEARCH'):
-        <div id="dashboard-search-bar" class="search-bar dashboard-search-bar" role="search" aria-label="Dashboard">
-          <form class="search-form">
-            <label for="dashboard-search-input">${_('Search Your Courses')}</label>
-            <div class="search-field-wrapper">
-              <input id="dashboard-search-input" type="text" class="search-field"/>
-              <button type="submit" class="search-button" title="${_('Search')}">
-                <span class="icon fa fa-search" aria-hidden="true"></span>
-              </button>
-              <button type="button" class="cancel-button" title="${_('Clear search')}">
-                <span class="icon fa fa-remove" aria-hidden="true"></span>
-              </button>
+            % for course_dir, errors in errored_courses.items():
+               <h3>${course_dir}</h3>
+                   <ul>
+                 % for (msg, err) in errors:
```

Comment:
Nit: the indentation looks off here.

Reply:
I don't like abbreviations, so can you use ``message`` and ``error``?

Reply:
I realize this isn't your code, but I don't think showing an error message is a good experience. It probably isn't localized, and isn't intended for display.